ACTUAL_TYPE_ERROR = "value passed to asserto({0}) must be of type {1} when calling {2} but it was {3}."